### PR TITLE
(GH-603, GH-607) Use source_list method to get sources

### PR DESCRIPTION
**Pull request by gep13**
- Rather than reading the configuration file directly
- This means that Chocolatey.Lib will do the work for filtering this sources list if/when required

### CODE REVIEW

**Comment by gep13** [comment]
Relates to #603

**Comment by gep13** [comment]
@ferventcoder I have made the change that was requested, and also took an initial pass at removing the multiple calls to Lets.GetChocolatey.

Let me know if you have any further comments.

File: Source/ChocolateyGui/Services/ChocolateyService.cs [hunk 1]
```diff
@@ -359,25 +355,25 @@ public Task<bool> IsElevated()
...
 
         public async Task<ChocolateySource[]> GetSources()
         {
             using (await Lock.ReadLockAsync())
             {
-                var config = await GetConfigFile();
-                return config.Sources.Select(_mapper.Map<ChocolateySource>).ToArray();
+                var sources = _configSettingsService.source_list(_choco.GetConfiguration());
+                var mappedSources = sources.Select(_mapper.Map<ChocolateySource>).ToArray();
+                return mappedSources;
```
Comment (ferventcoder) [inline Source/ChocolateyGui/Services/ChocolateyService.cs]: 👍

**Comment by gep13** [comment]
Also relates to #607

**Comment by gep13** [comment]
@ferventcoder thanks for merging.

**Pull request merged**